src/library/trackmodel.h

```diff
@@ -29,26 +29,27 @@ class TrackModel {
     // These enums are the bits in a bitvector. Any individual column cannot
     // have a value other than 0, 1, 2, 4, or 8!
     enum Capabilities {
-        TRACKMODELCAPS_NONE              = 0x00000,
-        TRACKMODELCAPS_REORDER           = 0x00001,
-        TRACKMODELCAPS_RECEIVEDROPS      = 0x00002,
-        TRACKMODELCAPS_ADDTOPLAYLIST     = 0x00004,
-        TRACKMODELCAPS_ADDTOCRATE        = 0x00008,
-        TRACKMODELCAPS_ADDTOAUTODJ       = 0x00010,
-        TRACKMODELCAPS_LOCKED            = 0x00020,
-        TRACKMODELCAPS_EDITMETADATA      = 0x00040,
-        TRACKMODELCAPS_LOADTODECK        = 0x00080,
-        TRACKMODELCAPS_LOADTOSAMPLER     = 0x00100,
-        TRACKMODELCAPS_LOADTOPREVIEWDECK = 0x00200,
-        TRACKMODELCAPS_REMOVE            = 0x00400,
-        TRACKMODELCAPS_RESETPLAYED       = 0x02000,
-        TRACKMODELCAPS_HIDE              = 0x04000,
-        TRACKMODELCAPS_UNHIDE            = 0x08000,
-        TRACKMODELCAPS_PURGE             = 0x10000,
-        TRACKMODELCAPS_REMOVE_PLAYLIST   = 0x20000,
-        TRACKMODELCAPS_REMOVE_CRATE      = 0x40000,
+        TRACKMODELCAPS_NONE              = 0x00000u,
```

Comment:
Is this a clang-format complaint format. Even though it looks good, we should not do it, because the next edit here will mess up everything due to the pre-comit hook.

Reply:
This will be changed in #2810 anyways